tianshou/policy/base.py

```diff
@@ -149,7 +151,7 @@ def compute_nstep_return(
         target_q_fn: Callable[[ReplayBuffer, np.ndarray], torch.Tensor],
         gamma: float = 0.99,
         n_step: int = 1,
-        rew_norm: bool = False
+        rew_norm: bool = False,
```

Comment:
typo I guess.

Reply:
A trailing comma is okay, though. Future modifications then will not include this comma.